tests.py

```diff
@@ -337,6 +337,15 @@ def test_enable_single_thread(self):
             'Segmentation fault',
             all_threads=False)
 
+    def test_enable_env_var(self):
+        self.check_fatal_error("""
+            import faulthandler
+            faulthandler._sigsegv()
```

Comment:
I would prefer a simple test checking if the module is enabled. See test_faulthandler.py in CPython:
https://hg.python.org/cpython/file/e4802f19eee4/Lib/test/test_faulthandler.py#l309